src/Orders.sol

```diff
@@ -5,103 +5,110 @@ import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 
 /// @notice Contract capable of processing fulfillment of intent-based Orders.
 abstract contract OrderDestination {
-    /// @notice Emitted when an swap order is fulfilled by the Builder.
-    /// @param originChainId - The chainId on which the swap order was submitted.
+    /// @notice Emitted when an Order output is fulfilled.
     /// @param token - The address of the token transferred to the recipient. address(0) corresponds to native Ether.
-    /// @param recipient - The recipient of the token.
     /// @param amount - The amount of the token transferred to the recipient.
-    event SwapFulfilled(
-        uint256 indexed originChainId, address indexed token, address indexed recipient, uint256 amount
-    );
+    /// @param recipient - The recipient of the token.
+    event OrderFulfilled(address indexed recipient, address indexed token, uint256 amount);
```

Comment:
why delete the param?

Reply:
the param was originally intended to tie the output to the order initiation for observability purposes, but it doesn't actually accomplish that - Orders don't actually have a nonce / unique identifier in our system, so the origin chain id is only directionally helpful.

still, in a world with multiple origin chains it could be good for general analytics.. 

but considering it while writing this PR, it seemed to me like 
- extra work for the Filler to input 
- no enforcement that the correct value is input
- extra gas cost to pass as calldata & emit
- not used by the contract at all 
- not used by the node at all

so it seemed it might be cleaner to just remove it

Reply:
okay, the tradeoff is that if there are ever multiple systems operating off this contract, they need to have a total ordering across them

Reply:
> okay, the tradeoff is that if there are ever multiple systems operating off this contract, they need to have a total ordering across them

hmm can you elaborate on this, i don't think i understand - regardless of how many systems operate on top of the contract, shouldn't the entire host chain block always be the most base level, mining before everything else?